grails-app/services/au/org/ala/merit/ProjectService.groovy

```diff
@@ -1808,10 +1808,11 @@ class ProjectService  {
         List documents = result?.documents ?: []
         if (documents) {
             documents = documents.collect {
-                Map resp = webService.getJson2(it.url)
```

Comment:
Why did you replace the getJson2 call with getJson?
The reason for the API change was that getJson produces an inconsistent response between a successful call and failed call (in particular if the response type is a List).  Json2 also uses a faster parser implementation.
An example of the untidy API is you've had to check for a null statusCode to indicate a successful call which is a bit untidy.

Reply:
I've committed the update to use getJson2() instead. I used the getJson() because of testing issue I am having with getJson2() but just noticed that my test data setup was incorrect.